packages/nimble-storybook/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@ni-private/nimble-storybook",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "description": "Storybook for the Nimble repo",
+  "scripts": {
+    "build": "tsc -p ./tsconfig.json && storybook build -o dist/storybook --webpack-stats-json",
+    "start": "storybook dev -p 6006",
+    "lint": "eslint .",
+    "format": "eslint . --fix"
+  },
+  "dependencies": {
+    "@microsoft/fast-react-wrapper": "^0.3.22",
+    "@ni/nimble-components": "^26.0.0",
+    "@ni/spright-components": "^0.0.2",
+    "@ni/nimble-tokens": "^6.13.2",
+    "@storybook/addon-a11y": "^8.0.4",
+    "@storybook/addon-actions": "^8.0.4",
+    "@storybook/addon-docs": "^8.0.4",
+    "@storybook/addon-essentials": "^8.0.4",
+    "@storybook/addon-interactions": "^8.0.4",
+    "@storybook/addon-links": "^8.0.4",
+    "@storybook/addon-webpack5-compiler-swc": "^1.0.2",
+    "@storybook/cli": "^8.0.4",
+    "@storybook/csf": "^0.1.2",
+    "@storybook/html": "^8.0.4",
+    "@storybook/html-webpack5": "^8.0.4",
+    "@storybook/manager-api": "^8.0.4",
+    "@storybook/theming": "^8.0.4",
+    "circular-dependency-plugin": "^5.2.0",
+    "js-beautify": "^1.15.1",
+    "remark-gfm": "^4.0.0",
+    "storybook": "^8.0.4",
+    "storybook-addon-pseudo-states": "^3.0.0",
+    "terser-webpack-plugin": "^5.3.10"
+  },
+  "devDependencies": {
+    "@chromatic-com/storybook": "^1.2.25",
+    "@ni-private/eslint-config-nimble": "^1.0.0",
+    "eslint-plugin-storybook": "^0.8.0",
```

Comment:
eslint configs should always be peerDep

Reply:
Why not devDep?

Reply:
See discussion https://github.com/ni/javascript-styleguide/issues/93